.circleci/config.yml

```diff
@@ -279,6 +279,7 @@ jobs:
       - run:
           name: 'End-to-End Tests'
           command: node build-system/pr-check/e2e-tests.js
+      - save_artifacts
```

Comment:
I'm guessing this should be `store_test_output`?

Reply:
Indeed it should, rebase got me